TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/auto/CompetitionAutonomous.java

```diff
@@ -29,15 +29,29 @@ public enum From {
         START, CONE_STACK
     }
 
+    public enum TeamState {
```

Comment:
Let's break this into two `enums` -- `TeamColor` and `TeamSide`

Reply:
got it, ill do it during break